docker/misk-web/misk-web

```diff
@@ -21,10 +21,7 @@ usage() {
 
   -l    'lint' run Prettier linting
 
-  -o    compute a md5 hash over the current directory and stores in .hash file
-
-  -m    compute a md5 hash over the current directory and stores in .hash file, creates .hashdiff file if there is an existing hash
-        exclude patterns: cachedUrls, lib, node_modules, .hash
+  -m    compute a md5 hash over the current directory and stores in .hash file
```

Comment:
Does it store it on its own, or do you have to redirect it? It looks like it's being redirected above.

Reply:
It automatically uses `tee` to store to `.hash` in the current directory but also outputs to standard out the hash for logging and so it can be redirected if the hash should go to another location.